docs/usage.md

```diff
@@ -1,66 +1,49 @@
 # nf-core/staramr: Usage
```

Comment:
Can you change to **phac-nml/staramrnf**

Reply:
53bd1b960ddf2860b0cb04c4fe931d4e98a0ef93